<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QWidget" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>更改密码</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/images/password.png</normaloff>:/image/images/password.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget{background-color: rgb(224, 224, 224);}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>260</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget{
	border-radius:5px;
	border:2px solid rgb(210, 210, 210);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="oldPasswordLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(34, 34, 34); 
font: 75 11pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>用户密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="oldPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="newPasswordLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(34, 34, 34); 
font: 75 11pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>更改密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="verifyPasswordLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(34, 34, 34); 
font: 75 11pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="verifyPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="finishButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="tradewidget.qrc"/>
 </resources>
 <connections/>
</ui>
